src/Sylius/Bundle/ApiBundle/spec/ApiPlatform/ConfigMergeManagerSpec.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace spec\Sylius\Bundle\ApiBundle\ApiPlatform;
+
+use PhpSpec\ObjectBehavior;
+
+class ConfigMergeManagerSpec extends ObjectBehavior
```

Comment:
```suggestion
final class ConfigMergeManagerSpec extends ObjectBehavior
```
in addition, missing license block and strict types - we should automate these things